tests/integration_tests/test_applications.py

```diff
@@ -655,10 +655,7 @@ def download_file(url):
 
     prepare_one_file("PSFcurve_data_v2.txt")
     prepare_one_file("MLTdata-preproduction.ecsv")
-    # TODO - temporary path until workflows/documentation PR is merged
-    download_file(
-        "https://raw.githubusercontent.com/gammasim/workflows/documentation/schemas/jsonschema.yml"
-    )
+    download_file("https://github.com/gammasim/workflows/blob/main/schemas/jsonschema.yml")
```

Comment:
Usually the links start with `raw`, it is not important here?

Reply:
You are right - asking the internet and it seems the majority (but not all) pages recommend using raw. Changed it. Will merge after tests are successful.